spec/fixtures/responses/mutations/default.json

```diff
@@ -0,0 +1,12 @@
+{
+  "status": 200,
+  "body": {
+    "data": {
+      "norms": {
+        "nodes": [
```

Comment:
What is this fixture for ?

Reply:
I have a mistake in freak_spec. I fix this